interview-questions.md

```diff
@@ -5,57 +5,73 @@ Answer the following questions. First, without external resources. Challenge you
 1. Why would you use the method super()?
 
   Your answer:
+  the super() method allows us to more quickly create class extensions without repeatedly entering constructors that would be used throughout all of them.
 
   Researched answer:
-
+  you would use the super method to access and call on functions from an object's parent.
 
 
 2. What is a virtual DOM?
 
   Your answer:
+  A virtual DOM (Document Object Model) off the top of my head, I think it's the model in which browsers see our data.
```

Comment:
Great answer. Even though you were not sure the answer, you answered honestly.  I would encourage you, while thinking of these questions as being in a real interview scenario, to answer honestly yet follow up with how/what you would do to find the correct answer. Bryan and the people you interview with interview practice will LOVE you for it :) Just sharing some cheat codes on that.